docs/helm/examples/with-ms-teams.md

```diff
@@ -6,7 +6,7 @@ grand_parent: Helm
 
 # ACS Helm Deployment with Microsoft Teams Connector
 
-The [Alfresco Microsoft Teams Connector](https://docs.alfresco.com/) enables
+The [Alfresco Microsoft Teams Connector](https://support.hyland.com/r/Alfresco/Alfresco-Process-Automation/7.17/Alfresco-Process-Automation/Model/Connectors/Microsoft-Teams-Connector) enables
```

Comment:
not the right one, can't find the equivalent of https://docs.alfresco.com/microsoft-teams/latest/

Reply:
The previous link pointed to the main doc page. Let's do the same here but to the new domain